src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -437,4 +437,10 @@
   <data name="DelegateInvocationCanBeSimplified" xml:space="preserve">
     <value>Delegate invocation can be simplified.</value>
   </data>
-</root>
+  <data name="AddParameterXmlDocumentation" xml:space="preserve">
+    <value>Add XML documentation for parameter {0}</value>
```

Comment:
I'd probably put the name in single quotes. I.e. `Add XML documentation for parameter '{0}'`